client/i18n/locales/english/intro.json

```diff
@@ -1356,15 +1370,22 @@
   },
   "upcoming-python": {
     "title": "Upcoming Python",
-    "intro": ["placeholder"],
+    "intro": [
+      "placeholder"
+    ],
```

Comment:
and so on

Reply:
I see. I never modified these lines, so I think my editor is "prettyfying" the code automatically. I'll get that fixed asap.